ompi/mpi/c/comm_create_from_group.c

```diff
@@ -81,8 +82,22 @@ int MPI_Comm_create_from_group (MPI_Group group, const char *tag, MPI_Info info,
     }
 
 
-    rc = ompi_comm_create_from_group ((ompi_group_t *) group, tag, &info->super, errhandler,
+    /*
+     * we use PMIx group operations to implement comm/intercomm create from group/groups.
+     * PMIx group constructors require a unique tag across the processes using the same
```

Comment:
Just a nit - it has to be unique across the PMIx session, not the server. In other words, if we are operating in a DVM and another `prun` executes a job that also utilizes PMIx groups, then the tags used by each job must be unique within that DVM and not just their local server. HTH.